### PR TITLE
Fabo/v 1220 unbonding transaction

### PR DESCRIPTION
**Pull request by faboweb**
added end unbonding tx

image

### CODE REVIEW

File: src/components/TmLiStakeTransaction/TmLiStakeTransaction.vue [hunk 1]
```diff
@@ -25,19 +25,25 @@ tm-li-transaction(:color="color" :time="transaction.time" :block="transaction.he
   template(v-if="unbonding")
     div(slot="caption")
       | Unbonded&nbsp;
-      template(v-if="transaction.undelegation")
-        b {{transaction.undelegation.amount}}
-        span &nbsp;{{transaction.undelegation.denom.toUpperCase()}}S
+      template(v-if="transaction.balance")
+        b {{transaction.balance.amount}}
+        span &nbsp;{{transaction.balance.denom.toUpperCase()}}S
```
Comment (jbibla) [inline src/components/TmLiStakeTransaction/TmLiStakeTransaction.vue]: are we still uppercasing things?

File: tests/unit/__snapshots__/TmBlock.spec.js.snap [hunk 2]
```diff
@@ -383,7 +383,8 @@ exports[`TmBlock.vue has the expected html structure 1`] = `
                 <p class="tm-li-tx__content__caption__title">
                   <div>
                     Unbonded&nbsp;
-                    <b>105</b><span>&nbsp;ATOMS</span><span>&nbsp;- a few seconds ago</span>
+                    <b>100</b><span>&nbsp;Shares</span>
```
Comment (jbibla) [inline tests/unit/__snapshots__/TmBlock.spec.js.snap]: i thought we were going to calculate the value of the shares in Atoms?

Comment (faboweb) [reply tests/unit/__snapshots__/TmBlock.spec.js.snap]: This is fallback logic. Atoms need to be calculated and passed in. But the transaction component also works with the plain TX.

File: src/components/TmLiStakeTransaction/TmLiStakeTransaction.vue (hunk 1 above)
Comment (faboweb) [reply src/components/TmLiStakeTransaction/TmLiStakeTransaction.vue]: Your call. I don't mind either.

**Pull request merged**